tools/storage/uri/src/main/java/org/projectnessie/storage/uri/StorageUri.java

```diff
@@ -70,6 +71,14 @@ private StorageUri(String scheme, String authority, String path) {
     this.path = path;
   }
 
+  public URI asURI() {
```

Comment:
I think, this could be removed with the other comments addressed?

Reply:
removed